drake/systems/Simulation.h

```diff
@@ -1,9 +1,15 @@
 #ifndef DRAKE_SYSTEMS_SIMULATION_H_
 #define DRAKE_SYSTEMS_SIMULATION_H_
 
-#include <thread>
 #include <chrono>
 #include <stdexcept>
+#include <thread>
+#include <iostream>
+#include <Eigen/Dense>
+
+#include <drake/core/Vector.h>
+
+using Drake::toEigen;
```

Comment:
Headers should not have using at global scope; it pollutes the global namespace.